include/bluetooth/bluetooth.h

```diff
@@ -1140,6 +1140,19 @@ int bt_le_per_adv_sync_create(const struct bt_le_per_adv_sync_param *param,
  */
 int bt_le_per_adv_sync_delete(struct bt_le_per_adv_sync *per_adv_sync);
 
+/**
+ * @brief Register periodic advertising sync callbacks.
+ *
+ * Adds the callback structure to the list of callback structures for periodic
+ * adverising syncs.
+ *
+ * This callback will be called for all periodic advertising sync activity,
+ * such as synced, terminated and when data is received.
+ *
+ * @param cb Callback struct. Must point to static memory.
```

Comment:
It doesn't have to be static though.
The note should perhaps be "Must point to memory that remains valid"

Should update bt_le_scan_cb_register, bt_conn_auth_cb_register and bt_le_conn_cb_register to match wording across these.

Reply:
I see you figured out where the text came from ;) 
But I agree on the new wording. Updated.

Reply:
That was quick :) Thanks.